sqlstore/src/main/resources/db/migration/V8000__ts_many_to_many.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE `TargetedSequencingTemp` (
+  `targetedSequencingId` bigint(20) NOT NULL AUTO_INCREMENT,
+  `alias` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `kitDescriptorId` bigint(20) NOT NULL,
```

Comment:
This column isn't needed even temporarily, so you could just not create it. Doesn't really matter though